.github/workflows/ai-fix.yml

```
@@ -0,0 +1,72 @@
+            echo "Strategy file not created. Falling back to basic instructions."
+            echo "Fix all issues mentioned in reviews.txt. Add JSDoc comments to all functions." > strategy.md
+          fi
+
+          # 3. Run aider with official flags
+          aider --model deepseek/deepseek-chat --message-file strategy.md --yes --no-check-update || {
+            echo "Aider execution failed"
+            exit 1
+          }
+
+      - name: Push Changes
+        run: |
```

actionlint (1.7.10)

[error] "github.head_ref" is potentially untrusted. avoid using it directly in inline scripts. instead, pass it through an environment variable. see https://docs.github.com/en/actions/reference/security/secure-use#good-practices-for-mitigating-script-injection-attacks for more details

(expression)